packages/flutter_tools/test/asset_bundle_package_fonts_test.dart

```diff
@@ -89,25 +96,38 @@ $fontsSection
       ..writeAsStringSync(font);
   }
 
-  // These tests do not use a memory file system because we want to ensure that
-  // asset bundles work correctly on Windows and Posix systems.
-  Directory tempDir;
-  Directory oldCurrentDir;
+  // TODO(dantup): This function exists in many tests, is there somewhere appropriate
```

Comment:
You can just create a file in the test directory, name it appropriately (do not end it with _test) and put all the logic there. We have done that before in other places, for example: https://github.com/flutter/flutter/blob/master/packages/flutter/test/widgets/semantics_tester.dart

Reply:
It was the "naming appropriately" bit I wasn't sure about. I've added a new file in `test/src/schema.dart` but it feels a bit only having two functions in it! LMK if you have a better idea - I've changed the other code that had copies of this (it's all the same - just making an empty schema because it's required to parse the pubspec) to use this one.